src/build123d/build_common.py

```diff
@@ -277,6 +278,7 @@ def faces(self, select: Select = Select.ALL) -> ShapeList[Face]:
         if select == Select.ALL:
             face_list = self._obj.faces()
         elif select == Select.LAST:
+            # TODO: move to dedicated builder?
```

Comment:
problem here is that this class does not have a last_faces and last_solids attribute

Reply:
True but some of the derived classes do.  Those that don't override this method with one that raises NotImplementedErrror. IIRC this was done to help with consistency in documentation.